.circleci/continue_config.yml

```diff
@@ -1462,6 +1504,28 @@ jobs:
       - run:
           name: Run tests
           command: DD_TRACE_AGENT_TIMEOUT=1000 <<# parameters.disable_runner_distributed_tracing >> DD_DISTRIBUTED_TRACING=false <</ parameters.disable_runner_distributed_tracing >> DD_TRACE_TEST_SAPI=<< parameters.sapi >> make << parameters.make_target >> RUST_DEBUG_SYMBOLS=1 PHPUNIT_OPTS="--log-junit test-results/php-composer/results.xml"
+      - when:
+          # codecov uploader only on amd64
+          condition:
+            and:
+              - equal: [ true, << parameters.coverage >> ]
+              - matches:
+                  pattern: "^[^.]+$"
+                  value: << parameters.resource_class >>
```

Comment:
We test ... that no dot is in the resource name because they're prefixed `arm.`... right.